llama.h

```diff
@@ -817,6 +817,9 @@ extern "C" {
     // Check if the token is supposed to end generation (end-of-generation, eg. EOS, EOT, etc.)
     LLAMA_API bool llama_token_is_eog(const struct llama_model * model, llama_token token);
 
+    // Identify if Token Id is a control token or a render-able token
+    LLAMA_API bool llama_token_is_control_token(const struct llama_model * model, llama_token token);
```

Comment:
No need for the `_token` suffix in the function name. Also, the API already tends to use the term "special" over "control" so better to call it:

```suggestion
    LLAMA_API bool llama_token_is_special(const struct llama_model * model, llama_token token);
```

Reply:
Okay, pushed a commit that drops the _token suffix